.github/workflows/release-pr.yml

```diff
@@ -0,0 +1,75 @@
+name: Open a release PR
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Version to release
+        required: true
+        type: string
+
+jobs:
+  make-release-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install cargo-release
+        uses: baptiste0928/cargo-install@v1
```

Comment:
Maybe we can use https://github.com/taiki-e/install-action ?

Reply:
`taiki-e/install-action` uses `cargo-binstall` and turns out that [`quickinstall` has `cargo-release`](https://github.com/alsuren/cargo-quickinstall/releases?q=cargo-release&expanded=true).

Reply:
That _would_ be beautifully circular, wouldn't it

Reply:
Yeah.